script.js

```diff
@@ -0,0 +1,58 @@
+let books = [
+  {
+    title: 'Uno Lorem',
+    author: 'Testero Testy',
+  },
+  {
+    title: 'Dos Lorem',
+    author: 'Testero Testy2',
+  },
+];
+
+function displayBooks() {
+  document.getElementById('book-list').innerHTML = '';
+  for (let i = 0; i < books.length; i += 1) {
+    document.getElementById('book-list').innerHTML += `
+  <li class="book-card">
+    <p>${books[i].title}</p>
+    <p>${books[i].author}</p>
+    <button id="remove${i}"class="remove" onclick="removeBook(this.id)">Remove</button>
+  </li>`;
+  }
+}
+
+function catchValue() {
+  const saveCollection = JSON.stringify(books);
+  localStorage.setItem('books', saveCollection);
+}
+/* eslint-disable no-unused-vars */
+function addBook() {
+  const newBook = {};
+  newBook.title = document.getElementById('title').value;
+  newBook.author = document.getElementById('author').value;
+  if (newBook.title !== '' && newBook.author !== '') {
+    books.push(newBook);
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+    catchValue();
+    displayBooks();
+  }
+}
+/* eslint-disable no-unused-vars */
+function removeBook(id) {
+  const bookDeleted = id.replace('remove', '');
+  books.splice(bookDeleted, 1);
+  catchValue();
+  displayBooks();
+}
```

Comment:
- [ ] I will suggest adding an `id` property to the books and then using `.filter()` inside this function instead of using the index, using the index can create issues in the future, kindly implement something like this:
    ```javascript
  // Book
  const firstBook = {
      id: 2,  // Generate the id's ramdomly
      title: Bad Boys,
      author: Vin Diesel,
  }

  // Use filter.() to remove the book
  function removeBook(id) {
    books = books.filter(book => book != id);
    catchValue();
    displayBooks();
  }
    ```

Reply:
This issue has been addressed well. Great job! :clap: :clap: